<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>4</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow
{
background-image: url(&quot;D:/Qt_practice/myGame/背景图片.jpg&quot;);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="beginButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>1070</y>
      <width>100</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#beginButton {
    border: 1px solid black;  /* 添加边框样式 */
    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>1011</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_1 
 {
 border: 1px solid black;  /* 添加边框样式 */    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>1011</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_2
 {
 border: 1px solid black;  /* 添加边框样式 */    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>275</x>
      <y>1011</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_3
 {
 border: 1px solid black;  /* 添加边框样式 */    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>1011</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_4
 {
 border: 1px solid black;  /* 添加边框样式 */    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>574</x>
      <y>1011</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_6
 {
 border: 1px solid black;  /* 添加边框样式 */    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>475</x>
      <y>1011</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_5
 {
 border: 1px solid black;  /* 添加边框样式 */    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>675</x>
      <y>1011</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_7
 {
 border: 1px solid black;  /* 添加边框样式 */    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>775</x>
      <y>1011</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_8
 {
 border: 1px solid black;  /* 添加边框样式 */    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_9">
    <property name="geometry">
     <rect>
      <x>875</x>
      <y>1011</y>
      <width>55</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>55</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_9
 {
 border: 1px solid black;  /* 添加边框样式 */    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="GameArea" name="widgetGameArea" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>900</width>
      <height>900</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>900</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="operatingLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>35</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>操作方式:</string>
    </property>
   </widget>
   <widget class="QLabel" name="operatingLabel_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>50</y>
      <width>601</width>
      <height>50</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>鼠标点击方格，再点击下方数字选中需要填入的数字</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reminderButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>1070</y>
      <width>100</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#reminderButton {
    border: 1px solid black;  /* 添加边框样式 */
    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>提示</string>
    </property>
   </widget>
   <widget class="QPushButton" name="newgameButton">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>1070</y>
      <width>120</width>
      <height>55</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>120</width>
      <height>55</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>120</width>
      <height>55</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#newgameButton {
    border: 1px solid black;  /* 添加边框样式 */
    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>新的游戏</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>61</width>
      <height>35</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>时间:</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>得分:</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>35</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>1070</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton {
    border: 1px solid black;  /* 添加边框样式 */
    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>排行榜</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_10">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>1070</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_10 {
    border: 1px solid black;  /* 添加边框样式 */
    border-radius: 10px;     /* 设置圆角 */
}</string>
    </property>
    <property name="text">
     <string>难度</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="musicBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>1130</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>音乐</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GameArea</class>
   <extends>QWidget</extends>
   <header>gamearea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
